README.adoc

```diff
@@ -114,7 +114,7 @@ $ minishift start
 . Log into the OpenShift cluster.
 +
 ----
-$ odo login -u developer -p developer
+$ odo login --username developer --password developer
```

Comment:
I don't like how we're using `--username` and `--password` here. Could we stick with `-u`? WDYT @kadel

Reply:
Well, expanding options in the commands is just to enable first time users to understand quickly. If this is too obvious we can change it back. Let me know what you think.

Reply:
`-u` and `-p` is quite obivious, even noob will understand, so we should change it back
I agree with @cdrage

Reply:
Reverted